Documentation/ApiOverview/Assets/Index.rst

```diff
@@ -16,7 +16,7 @@ tags.
 
 There are multiple ways to add assets to the :php:`PageRenderer` in TYPO3.
 For configuration options via TypoScript (usually used for the main theme files),
-see the :ref:`TypoScript Reference<t3tsref:setup-page-includecss-array`. In extensions,
+see the :ref:`TypoScript Reference <t3tsref:setup-page-includecss-array>`. In extensions,
```

Comment:
I wonder if we can use the regular `ref` here ...

Reply:
Why? I rendered it and the link is displayed correctly now.

Reply:
For consistency. We use `:ref:` everywhere so at least I wonder why we do something different here.

Reply:
I don't understand? I am using :ref:?

    :ref:`TypoScript Reference <t3tsref:setup-page-includecss-array>`

Reply:
Yes, you are right. It's Monday morning.